### PR TITLE
Fix security vulnerabilities

### PR DESCRIPTION
**Pull request by andriisoldatenko**
**Which issue this PR fixes**:

fix https://github.com/astronomer/issues/issues/857
fix https://github.com/astronomer/issues/issues/743

**Summary of changes**:

**Which images are updated by this PR?**:

- [x] alertmanager
- [x] curator
- [x] elasticsearch-exporter
- [x] elasticsearch
- [x] fluentd
- [x] grafana
- [x] keda-metrics-adapter
- [x] keda
- [x] kibana
- [x] nginx-es
- [x] redis
- [x] registry
- [x] statsd-exporter
(generated using `git diff --name-status master`)

**Checklist** (required)

Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.

- [ ] version.txt is updated in the changed image(s)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -511,7 +640,7 @@ commands:
             # Make this environment variable available to following steps
             echo "export TAG=${TAG}" >> $BASH_ENV
   clair-scan:
-    description: "Vulnerability scan a Docker image"
+    description: "Clair: Vulnerability scan a Docker image"
```
Comment (ashb) [inline .circleci/config.yml]: Is it worth keeping clair and trivy running?

(Let me guess: they find different things)

File: redis/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM redis:4.0.14-alpine3.10
+FROM redis:4.0.14-alpine3.11
```
Comment (ashb) [inline redis/Dockerfile]: We should update our Airflow images too -- they are based of 3.10 -- and annoyingly have `alpine-3.10` in the name.

Perhaps for those a rebuild against latest alpine3.10 is enough.

File: .circleci/config.yml (hunk 1 above)
Comment (andriisoldatenko) [reply .circleci/config.yml]: correct.

Comment (andriisoldatenko) [reply .circleci/config.yml]: https://github.com/astronomer/issues/issues/743#issuecomment-591687980

**Comment by vishwas-astro** [comment]
@ashb we will be running the entire test suite as part of 0.13 testing on staging anyway in addition to whatever testing @andriisoldatenko does in his local environment and in the dev cluster.

File: keda/Dockerfile [hunk 3]
```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.io/kedacore/keda:1.2.0
+FROM docker.io/kedacore/keda:master
```
Comment (sjmiller609) [inline keda/Dockerfile]: Let's pin it

**Comment by andriisoldatenko** [comment]
@ashb @schnie PTAL

File: .circleci/config.yml [hunk 4]
```diff
@@ -385,6 +490,50 @@ jobs:
...
+            # trivy doesn't support busybox scanning right now: https://github.com/aquasecurity/trivy/issues/397
+            # TODO: @andriisoldatenko patch trivy to add busybox
+            MATCHES=$(echo "$output" | grep -i 'unknown os' | wc -l)
+            if [[ ${MATCHES} -eq 1 ]]; then
+               echo "Skipping trivy scan because of unsupported OS"
+               exit 0
+            fi
+            echo "${output}"
+            exit ${exit_code}
+      - restore_cache:
```
Comment (ashb) [inline .circleci/config.yml]: Itym save cache here

File: keda-metrics-adapter/Dockerfile [hunk 5]
```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.io/kedacore/keda-metrics-adapter:1.2.0
+FROM docker.io/kedacore/keda-metrics-adapter:master
```
Comment (ashb) [inline keda-metrics-adapter/Dockerfile]: I don't think we should be using master/a moving target

Comment (schnie) [reply keda-metrics-adapter/Dockerfile]: KEDA doesn't have an updated version with the fixes yet?

Comment (andriisoldatenko) [reply keda-metrics-adapter/Dockerfile]: fixed.

File: .circleci/config.yml (hunk 4 above)
Comment (andriisoldatenko) [reply .circleci/config.yml]: fixed.

**Comment by andriisoldatenko** [comment]
@sjmiller609 what's best version to bump in `version.txt`?

**Comment by andriisoldatenko** [comment]
@ashb @schnie PTAL

File: keda/Dockerfile (hunk 3 above)
Comment (schnie) [reply keda/Dockerfile]: @andriisoldatenko @dimberman what do you guys think here?

Comment (schnie) [reply keda/Dockerfile]: oh wait, outdated.. nvm

**Comment by schnie** [comment]
Everything looks good to me. My only question is how we want to version this stuff. Should we bump all these `version.txt`s to 0.13.0 for now? Maybe we switch to actual software versions later?

**Comment by andriisoldatenko** [comment]
>0.13.0
sgtm

**Pull request merged**